src/storage/builder.rs

```diff
@@ -29,10 +29,19 @@ use super::prelude::*;
 /// [`max_data_in_blob`]: struct.Builder.html#method.max_data_in_blob
 /// [`blob_file_name_prefix`]: struct.Builder.html#method.blob_file_name_prefix
 /// [`key_size`]: struct.Builder.html#method.key_size
-#[derive(Default, Debug)]
+#[derive(Debug)]
 pub struct Builder {
     config: Config,
-    ioring: Option<Rio>,
+    iodriver: IoDriver,
+}
+
+impl Default for Builder {
+    fn default() -> Self {
+        Self {
+            config: Config::default(),
+            iodriver: IoDriver::new_sync(),
```

Comment:
This assumes, that `IoDriver::new_sync()` is cheap. I'd prefer to declare field with option: `iodriver: Option<IoDriver>` and set it here to `None`. In this way there is a guaranty, that initialization of builder will always be cheap